client-react/src/pages/app/app-settings/SettingSourceColumn.tsx

```diff
@@ -3,20 +3,16 @@ import React, { useContext } from 'react';
 import { useTranslation } from 'react-i18next';
 import { defaultCellStyle } from '../../../components/DisplayTableWithEmptyMessage/DisplayTableWithEmptyMessage';
 import { ThemeContext } from '../../../ThemeContext';
-import { keyVaultIconStyle, sourceTextStyle } from './AppSettings.styles';
-import { KeyVaultReferenceSummary } from './AppSettings.types';
+import { IconStyle, sourceTextStyle } from './AppSettings.styles';
```

Comment:
If `IconStyle` renamed to `iconStyle`:
```suggestion
import { iconStyle, sourceTextStyle } from './AppSettings.styles';
```